test/server/server_test.cc

```diff
@@ -521,8 +521,10 @@ class ServerStatsTest
 
   void flushStats() {
     if (manual_flush_) {
-      server_->flushStats();
```

Comment:
just split this into two tests so we continue to directly test server_->flushStats()